src/chapters/introduction/why-learn-to-code.rst

```diff
@@ -1,2 +1,28 @@
 Why Learn To Code?
 ==================
+
+How many times do you use a computer in a day?
+Maybe you use one to check email and social media, to watch TV, and to even set an alarm for the next day.
+Computers and technology are ubiquitous in our society. 
+
+With the rise of technology and computers, coding has risen as well.
+At its most basic level, coding is how humans communicate with computers.
+With code, humans tell computers to complete specific tasks and store specific information.
+Many would argue that due to the prevalence of computers, learning to code is vital to living in the 21st century.
+Writing and reading code is becoming a new form of literacy for today's world.
+
+As our needs change, technology changes to meet them.
+When we needed a way to talk to each other over long distances, we got phones.
+As our needs to communicate changed, our phones became portable.
+Then, we gained the ability to use our phones to send quick written messages to each other.
+
+The technical skills required to make the phones of 20 years ago are not the same skills required to make a phone today.
+A career as a technologist, specifically as a programmer, is one of lifelong learning. 
+
+Learning to code is not only valuable and challenging, it is also fun.
+Every "EUREKA!" moment is inspiration to keep going forward and to learn new things.
+You may find that some concepts are difficult to understand, but those are going to be the same concepts that you will be most proud of yourself for mastering.
```

Comment:
Clarity edit (+ growth mindset): "You may find some concepts difficult to understand at first, but these will also be the skills you take the most pride in mastering."

Reply:
Thanks!